guide/ops/troubleshooting/increase-system-resource-limits.md

```diff
@@ -4,6 +4,16 @@ title: Increase System Resource Limits
 toc: /guide/toc.json
 ---
 
+If you encounter the following error:
+
+    Caused by: java.io.IOException: Too many open files
+            at java.io.UnixFileSystem.createFileExclusively(Native Method)[:1.8.0
+
+Please check limit for opened files `cat /proc/sys/fs/file-max` and increase it.
```

Comment:
Well the missing `the` in `Please check limit` is definately required for proper english, if you want to keep `cat` in there, that's fine.